pkg/operation/botanist/secrets.go

```diff
@@ -91,10 +91,6 @@ func (b *Botanist) lastSecretRotationStartTimes() map[string]time.Time {
 			rotation[v1beta1constants.SecretNameSSHKeyPair] = shootStatus.Credentials.Rotation.SSHKeypair.LastInitiationTime.Time
 		}
 
-		if shootStatus.Credentials.Rotation.Observability != nil && shootStatus.Credentials.Rotation.Observability.LastInitiationTime != nil {
-			rotation[v1beta1constants.SecretNameObservabilityIngressUsers] = shootStatus.Credentials.Rotation.Observability.LastInitiationTime.Time
-		}
-
```

Comment:
Why did you remove this? We still want the credentials to be rotated when the `gardener.cloud/operation=rotate-observability-credentials` is annotated, don't we?